RevenueCat/Scripts/PurchasesWrapperiOS.cs

```diff
@@ -6,12 +6,13 @@
 public class PurchasesWrapperiOS : IPurchasesWrapper
 {
     [DllImport("__Internal")]
-    private static extern void _RCSetupPurchases(string gameObject, string apiKey, string appUserId, bool observerMode, string userDefaultsSuiteName);
+    private static extern void _RCSetupPurchases(string gameObject, string apiKey, string appUserId, bool observerMode,
+                                                 bool usesStoreKit2IfAvailable, string userDefaultsSuiteName);
 
-    public void Setup(string gameObject, string apiKey, string appUserId, bool observerMode,
+    public void Setup(string gameObject, string apiKey, string appUserId, bool observerMode, bool usesStoreKit2IfAvailable,
         string userDefaultsSuiteName, bool useAmazon, string dangerousSettingsJson)
     {
-        _RCSetupPurchases(gameObject, apiKey, appUserId, observerMode, userDefaultsSuiteName);
+        _RCSetupPurchases(gameObject, apiKey, appUserId, observerMode, usesStoreKit2IfAvailable, userDefaultsSuiteName);
```

Comment:
I didn't change it in this PR but we are not passing the `dangerousSettingsJson` parameter down to the iOS SDK it seems (though I guess we would need to parse it before). Should we do that? @aboedo.  I can tackle it in a different PR if so.

Reply:
oops
yeah, let's do it as a separate PR